kong-0.8.1-0.rockspec

```diff
@@ -35,7 +35,8 @@ dependencies = {
   "lua-llthreads2 ~> 0.1.3-1",
   "luacrypto >= 0.3.2-1",
   "luasyslog >= 1.0.0-2",
-  "lua_pack ~> 1.0.4-0"
+  "lua_pack ~> 1.0.4-0",
+  "luaossl ~> 20151221-0"
```

Comment:
I will look into it. I agree it would be better/easier not to introduce any more dependencies.